File: skills/playfetch/playfetch.py
#!/usr/bin/env python3
"""playfetch - загрузка страниц через Playwright"""
import sys
from playwright.sync_api import sync_playwright

if len(sys.argv) < 2:
    print("Usage: playfetch <url>", file=sys.stderr)
    sys.exit(1)

url = sys.argv[1]

with sync_playwright() as p:
    browser = p.chromium.launch(headless=True)
    page = browser.new_page()
    page.goto(url, timeout=30000)
    print(page.content())
    browser.close()

File: skills/playfetch/playfetch.js
#!/usr/bin/env node
/** playfetch — загрузка страниц через Playwright */
const { chromium } = require('playwright');

async function main() {
    if (process.argv.length < 3) {
        console.error('Usage: node playfetch.js <url>');
        process.exit(1);
    }

    const url = process.argv[2];

    const browser = await chromium.launch({ headless: true });
    const page = await browser.newPage();
    await page.goto(url, { timeout: 30000 });
    const content = await page.content();
    console.log(content);
    await browser.close();
}

main().catch(e => {
    console.error(e);
    process.exit(1);
});
